internal/rollup/rollup_bundle.bzl

```diff
@@ -235,16 +268,81 @@ def run_sourcemapexplorer(ctx, js, map, output):
       ],
   )
 
+def _generate_code_split_entry(ctx, folder, output):
+  """Generates a SystemJS boilerplate/entry point file.
+
+  See doc for additional_entry_points for more information.
+
+  Args:
+    ctx: bazel rule execution context
+    folder: the folder name with the bundled chunks
+    output: the file to generate
+  """
+  build_file_prefix = "/".join(ctx.build_file_path.split("/")[:-1]) + "/"
```

Comment:
would be nice to have a comment here describing an example ctx.build_file_path to see what this logic is meant to transform it to

Reply:
done. also when I was writing the comment I realized the logic was wrong as well. thanks!